traffic_ops/app/db/migrations/2023082308560644_server_update_status.up.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+ALTER TABLE public.server
+  ADD COLUMN IF NOT EXISTS config_update_failed boolean DEFAULT false,
+  ADD COLUMN IF NOT EXISTS revalidate_update_failed boolean DEFAULT false;
```

Comment:
Since these have default values, `NULL` values have no valid use or meaning, and scanning into a non-reference struct field would fail for `NULL` values, do you want to make these columns `NOT NULL`?